Add aspect intro (#70)
MiniFate/MiniFate: _chapters/05-aspects-and-fate-points.md

{% include macros.html %}

> An Aspect is a short phrase written about the in-game world by a player or the

Reviewer: short *improvised* phrase? short phrase *written on the fly*? seems worth emphasizing that there's no lookup table for aspects. You just write them down

GM. They are how the narrative interacts with the mechanics of the game,
allowing what makes a good story to drive what happens when the dice are
rolled. Aspects are how the players and the GM come to an agreed upon
understanding of the characters, the setting, and the current scene.

Some Aspects are attached to characters, allowing the player or GM to
express in their own words what makes the character unique and interesting.
Other Aspects refer to the world around the characters, or the story taking

> place in it; an Aspect can be a detail within the current scene, a summary of

Reviewer: ```suggestion
place in it. An Aspect can be a detail within the current scene, a summary of
```

the overarching plot arc, or literally any other phrase that the players or GM
find pertinent.

Aspects have three roles, each of which is crucial to the game:

> - **Aspects are always true**: They dictate what sort of play makes sense and

Reviewer: I think the jump from "aspects are always true" to "pyromancer shoots fireballs not crossbows" is tenuous. Maybe **Aspects are how the table judges what makes sense**?

Author: I think the first intro sentence of the bullet point covers this, do you?

Maybe **Aspects are truths about the world**? Hmm... I think I see your point. **Truth** and "Therefore you may only..." doesn't connect cleanly.

  what form a skill takes. If a character is ~~Grappled~~, it would not make
  sense for them to run away, swing a weapon, or do anything that requires
  free movement. If a character is a ~~Pyromancer of the Dancing Flame~~ then
  their `Shoot` attack will probably take the form of a conjured fireball; and
  they would be absolutely helpless if asked to pick up a crossbow.
- **Aspects can be invoked for a bonus on a roll**: If the dice say that

>   ~~Sherwood Forest's Greatest Archer~~ is about to miss a tough shot, the

Reviewer: ```suggestion
  ~~Sherwood's Greatest Archer~~ is about to miss a tough shot, the
```

  player can spend a fate point to get a bonus on the roll.
- **Aspects can be compelled to create engaging plot twists**: Twists and

>   turns make an engaging story; MiniFate encourages plot twists be

Reviewer: ```suggestion
  turns make an engaging story; MiniFate encourages plot twists by
```

>   turns make an engaging story; MiniFate encourages plot twists be

Reviewer: Something something make sure players are having a good time even if their characters aren't
